firmware/application/apps/ui_adsb_tx.cpp

```diff
@@ -296,6 +296,8 @@ void ADSBTxView::generate_frames() {
 void ADSBTxView::start_tx() {
     generate_frames();
 
+    /* Already tested , It is better to keep original condition, not activating  "transmitter_model.set_baseband_bandwidth(28'000'000)"
```

Comment:
This would only be the original condition on boot. Once another app has set up the registers, won't they be incorrect? Also, "enable" will always set the bandwidth to the value on the tx_model. Should there be a way to "reset" back to the mode that doesn't have the LPF enabled?

Reply:
I had exactly the same concerns as you , But I already tested that after Mic App , or any other App that is settings it to 1M75 , that ADSB TX App is always sending a broad bandwith , and his spectrum is higher than the maximum TX LPF 28M.

Then my current understanding is that all Tx App is calling  MAX2837::init() , and that is disabling the LFT  (Address 2, LPF Register 1 (Default = 1F4HEX)-
But I will double check about it , Let's leave that point open .

Reply:
It's a nice PR. I can't wait to have it in next.
Everything that is making the signal better is good :-)

Reply:
After deep checking and investigation . 
I could conclude that it is better to activate for ADSB TX  the TX LPF 6 Mhz , then that problem is solved.  See extended  attached comments below .